.rubocop.yml

```diff
@@ -74,6 +74,10 @@ Naming:
 
 #  Style stuff
 #
+Style/ArgumentsForwarding:
+  # disabled because this breaks on Ruby < 3.2
```

Comment:
mmh, I'm not sure if this is really needed, cause expecting ruby > 3.0 supports it

Reply:
See https://github.com/ruby-grape/grape-entity/actions/runs/7191884680

Anonymous args/kwargs forwarding was added in Ruby 3.2.

Reply:
BTW, it seems that a better solution is to change [this line](https://github.com/ruby-grape/grape-entity/blob/master/.rubocop.yml#L8) to 3.0.

It means to designate the _minimum_ Ruby version that should be supported by the code (so that Rubocop wouldn't propose features that emerged in later versions).

Reply:
no not really, want to support the future not the past

Reply:
@LeFnord The [exact meaning](https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version) of this field is to set the _minimum_ version that the codebase is intended to support. If you have a codebase that needs to support 3.0 and 3.1, and you set this field to 3.2, then Rubocop will suggest to you various Ruby features that _only_ work on 3.2 and make the code broken on 3.1 (the exact problem that happened in this PR).

Reply:
for compatibility we have a test matrix, rubocop is used for actual an future dev … but yeah, it seems, it makes no sense to discus